docs/extend/extend-apiml/api-mediation-caching-service.md

```diff
@@ -0,0 +1,133 @@
+# Using the Caching Service
```

Comment:
@nannanli this `extend-apiml/api-mediation-catching-service` & `extend-apiml/api-mediation-redis` needs to be referenced in the [sidebar.js](https://github.com/zowe/docs-site/blob/zowe-docs-v2/sidebars.js#L178).